site/de/index.html

```diff
@@ -0,0 +1,664 @@
+ImmutableMap&lt;String, String&gt; immutableMap =
+  Maps.immutable.with("key1", "value1", "key2", "value2", "key3", "value3");
+ImmutableMultimap&lt;String, String&gt; immutableMultimapWithList =
+  Multimaps.immutable.list.with("key1", "value1-1", "key1", "value1-2", "key2","value2-1");
+ImmutableBiMap&lt;String, String&gt; immutableBiMap =
+  BiMaps.immutable.with("key1", "value1", "key2", "value2", "key3", "value3");</pre>
+                        </div>
+                        <div class="tab-pane fade" id="primitives">
+                            <pre  class="prettyprint lang-java">
+//Veränderliche und unveränderliche Listen, Sets, Bags, Stacks und Maps
+//sind verfügbar für alle 8 primitiven Datentypen
+MutableIntList intList =
+  IntLists.mutable.of(1, 2, 3);
+MutableLongList longList =
+  LongLists.mutable.of(1L, 2L, 3L);
+MutableCharList charList =
+  CharLists.mutable.of('a', 'b', 'c');
+MutableShortList shortList =
+  ShortLists.mutable.of((short)1, (short)2, (short)3);
+MutableByteList byteList =
+  ByteLists.mutable.of((byte)1, (byte)2, (byte)3);
+MutableBooleanList booleanList =
+  BooleanLists.mutable.of(true, false);
+MutableFloatList floatList =
+  FloatLists.mutable.of(1.0f, 2.0f, 3.0f);
+MutableDoubleList doubleList =
+  DoubleLists.mutable.of(1.0, 2.0, 3.0);
+
+//Zahlenreichen können mit IntInterval erstellt werden
+IntInterval oneTo10 =
+  IntInterval.fromTo(1, 10); // ints von 1 bis 10
+// [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+IntInterval oneTo10By3 =
+  IntInterval.fromToBy(1, 10, 3); // ints von 1 bis 10 in 3 Sprüngen
+// [1, 4, 7, 10]
+IntInterval oddsFrom1To10 =
+  IntInterval.oddsFromTo(1, 10); // ungerade ints von 1 bis 10
+// [1, 3, 5, 7, 9]
+IntInterval evensFrom1To10 =
+  IntInterval.evensFromTo(1, 10); // gerade ints von 1 bis 10
+// [2, 4, 6, 8, 10]</pre>
+                        </div>
+                    </div>
+                    <!-- /.tab-content -->
+                </div>
+                <!-- /.col -->
+            </div>
+            <!-- /.row -->
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Container Types -->
+
+    <a class="anchor" name="memory"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-6 col-lg-6 col-sm-push-6 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Reduziert den Speicherverbrauch<br>mit speichereffizienten Daten-Containern.</h2>
+                    <p class="lead">Eclipse Collections bietet effiziente Implementierungen von Sets und Maps, sowie Primitive Collections. </p>
+                </div>
+                <div class="col-lg-pull-6 col-lg-6 col-sm-pull-6 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#set" data-toggle="tab">Sets</a></li>
```

Comment:
No image displayed here? Nothing happens when clicking on the tabs.

Reply:
The images will be rendered once the final index.html is merged. You can assume that it will be rendered. 😄